drivers/lm75/lm75.c

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_lm75
+ *
+ * @{
+ * @file
+ * @brief       Driver for the LM75 temperature sensor.
+ *
+ * A general driver for the lm75 temperature sensor including support for the
+ * lm75a and tmp1075 sensors as well.
+ *
+ * @author      Vitor Batista <vitor.batista@ml-pa.com>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "lm75.h"
+#include "lm75_regs.h"
+#include "lm75_params.h"
+#include "log.h"
+#include "kernel_defines.h"
+#include "periph/gpio.h"
+#include <stdint.h>
+#include <byteorder.h>
+#include "xtimer.h"
+
+#define I2C_BUS                 dev->lm75_params.i2c_bus
+#define I2C_ADDR                dev->lm75_params.i2c_addr
+#define LM75_SHUTDOWN_MODE      0x01
+#define LM75_OS_POLARITY        0x04
```

Comment:
Those are both for the CONFIG register right?

```suggestion
#define LM75_CONFIG_SHUTDOWN        0x01
#define LM75_CONFIG_OS_POLARITY     0x04
```

Reply:
They are, but mostly the CONFIG prefix is used just for compatibility with the Kconfigs.

Reply:
KConfig prefixes the whole define with `CONFIG_`.

I'm asking you to prefix the define with `LM75_CONFIG_` as it is related to the LM75 CONFIG register.